pages/index.js

```diff
@@ -1,9 +1,8 @@
 import Head from 'next/head';
-import Highlight from 'react-highlight.js';
 
 import { hooks } from '../hooks';
-
-const Code = ({ children }) => <Highlight language='javascript' className='code' children={children} />
+import Navbar from "../components/navbar/navbar";
```

Comment:
```suggestion
import Navbar from '../components/Navbar/Navbar';
```

Reply:
add a index.js file in Navbar folder. There write this code

```js
export * from './Navbar';
```

This way you don't have to write `import from 'components/Navbar/Navbar'` you can just `import from 'components/Navbar'`